eng/common/instructions/azsdk-tools/validate-service-label.instructions.md

```diff
@@ -6,22 +6,44 @@ tools: ['azsdk_check_service_label', 'azsdk_create_service_label']
 ## Goal
 Validate service label exists or create new one for SDK release process.
 
-## Step 1: Get Service Label
-Ask user for their service label. If none provided, explain that a valid service label is required for SDK release.
+## Step 1: Provide Information
+
+Provide the following information about the importance of service labels:
+
+"Before your SDK is released, your service must have a valid service label. Service labels enable automatic owner assignment and notifications across the Azure SDK ecosystem.
+
+When properly configured, service labels automatically:
+
+- Notify service owners when issues are filed against their SDK
+- Add appropriate reviewers to pull requests
+- Connect code changes to the right team members through CODEOWNERS integration
+
+Without a valid service label, the process to identify the correct service owners for issues and code reviews becomes manual and inefficient."
+
+## Step 2: Get Service Label
+
+Ask user for their service label. If they don't know their service label provide guidance:
+
+- Check out the [Common Labels CSV](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/data/common-labels.csv) file and look for a row who's first column contains your service's product name.
+
+If they don't have a service label - go to Step 3 for new service label
+
+## Step 3: Validate Label
 
-## Step 2: Validate Label
 Use `azsdk_check_service_label` to check status:
 
 - **Exists**: Success - user can proceed with next steps in SDK release process
 - **InReview**: Label pending approval - user can proceed (will be available once merged)
 - **DoesNotExist**: Go to Step 3 to create new label
 - **NotAServiceLabel**: Label exists but it is not a service label - go to Step 3 for new service label
 
-## Step 3: Create New Service Label
+## Step 4: Create New Service Label
```

Comment:
@mariari -- The rules for service labels are in here. The suggested label from our chat thread doesn't follow these instructions.

Reply:
@maririos for attn in case my edit of the above message doesn't generate a notification :P... I used the wrong github alias in the last message originally